src/main/java/com/google/devtools/build/lib/bazel/commands/SyncCommand.java

```diff
@@ -313,4 +320,39 @@ public Object getResolvedInformation() {
       }
     };
   }
+
+  @Nullable
+  private Iterable<Rule> getRules(OptionsParsingResult options, WorkspaceFileValue fileValue, CommandEnvironment env){
```

Comment:
NIT it's a bit unclear what `fileValue` is, how about renaming it to `workspace` or `workspaceFileValue`?